### PR TITLE
Add Duration rounding change to release note

### LINKED ISSUE #100480: std::time::Duration::from_secs_f64 behavior changed in 1.63.0
- ~Related release note: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#libraries-3~
- ~Related PR: #90247~

~Though the PR describes the change is related to improve its truncation better, the changed behavior is rounding up which was down before.~

EDIT: The upper changes and issue [related to 1.60](https://github.com/rust-lang/rust/issues/100480#issuecomment-1213815101). It is not related to this issue.

I want to check if this is expected change or regression.

### Code

I tried this code:

```rust
use std::time::Duration;

fn main() {
    let f: f64 = 1.0020030005;
    let d = Duration::from_secs_f64(f);
    println!("{d:?}");
}
```

I expected to see this happen: *explanation*
Until 1.62.1,
```
1.002003s
```

Instead, this happened: *explanation*
In 1.63.0,
```
1.002003001s
```

### Version it worked on

It most recently worked on: Until 1.62.1

### Version with regression

`rustc --version --verbose`:
```
rustc 1.63.0 (4b91a6ea7 2022-08-08)
binary: rustc
commit-hash: 4b91a6ea7258a947e59c6522cd5898e7c0a6a88f
commit-date: 2022-08-08
host: aarch64-apple-darwin
release: 1.63.0
LLVM version: 14.0.5
```

### PR DESCRIPTION
**Pull request by youknowone**
one of possible fix of #100480

fixes https://github.com/rust-lang/rust/issues/100480

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Libraries
 - [Extend `ptr::null` and `null_mut` to all thin (including extern) types.][94954]
 - [`impl Read and Write for VecDeque<u8>`.][95632]
 - [STD support for the Nintendo 3DS.][95897]
+- [Use rounding in float to Duration conversion methods.][96051]
```
Comment (Mark-Simulacrum) [inline RELEASES.md:30]: Can we also add this to the future compat section below?

Comment (youknowone) [reply RELEASES.md:30]: @Mark-Simulacrum is this what you meant?
https://github.com/rust-lang/rust/pull/100482/commits/cee573f32a1927ba938a29343bab2121136c5b28

**Comment by Mark-Simulacrum** [comment]
Yes, r=me with commits squashed.

**Pull request merged**